### PR TITLE
Add Curiefense

### PR DESCRIPTION
**Pull request by jdorfman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -357,6 +357,7 @@ Cloud Native is a behavior and design philosophy. At its essence, any behavior o
 - [checkov](https://github.com/bridgecrewio/checkov/) - A static analysis tool for infrastructure as code - to prevent misconfigs at build time.
 - [clair](https://github.com/quay/clair) - Vulnerability Static Analysis for Containers.
 - [cost-model](https://github.com/kubecost/cost-model) - Cross-cloud cost allocation models for workloads running on Kubernetes.
+- [curiefense](https://github.com/curiefense/curiefense) - Adds a broad set of automated web security tools to Envoy.
```
Comment (rootsongjc) [inline README.md:360]:
```suggestion
- [curiefense](https://github.com/curiefense/curiefense) - Curiefense is a unified, open source platform protecting cloud native applications.
```

Comment (jdorfman) [reply README.md:360]: That works. I was afraid it was too markety for this list. Thanks @rootsongjc!

Comment (rootsongjc) [reply README.md:360]: Just keep it the same with the description on the repo's homepage.

Comment (jdorfman) [reply README.md:360]: @rootsongjc LMK if there is anything else you need from me.

**Review by rootsongjc — APPROVED** [review]
LGTM

**Pull request merged**